libraries/testing/tester.cpp

```diff
@@ -374,8 +374,8 @@ namespace eosio::testing {
       [[maybe_unused]] auto block_start_connection = control->block_start().connect([](block_num_type num) {
          // only block number is signaled, in forking tests will get the same block number more than once.
       });
-      [[maybe_unused]] auto accepted_block_header_connection = control->accepted_block_header().connect([this](const block_signal_params& t) {
-            const auto& [block, id] = t;
+      [[maybe_unused]] auto accepted_block_header_connection = control->accepted_block_header().connect([&](const block_signal_params& t) {
```

Comment:
Why the change from `[this]` to `[&]` ?

Reply:
It gets rid of the second warning. `warning: lambda capture 'this' is not used`. I don't think we can use `[[maybe_unused]]` in capture list? If you have preference for another approach I'm totally open to it